server/tests/io/feed_parsers/ritzau_test.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2024 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import os
+import settings
+
+from superdesk import config
+from superdesk.etree import etree
+from superdesk.tests import TestCase
+
+from borsen.io.feed_parsers.ritzau import RitzauFeedParser
+
+
+class BaseRitzauTestCase(TestCase):
+
+    filename = "ritzau.xml"
+
+    def setUp(self):
+        super().setUp()
+        # settings are needed in order to get into account NITF_MAPPING
+        for key in dir(settings):
+            if key.isupper():
+                setattr(config, key, getattr(settings, key))
+
+        self._run_parse()
+
+    def _run_parse(self):
+        dirname = os.path.dirname(os.path.realpath(__file__))
+        fixture = os.path.normpath(os.path.join(dirname, "../fixtures", self.filename))
+        provider = {"_id": "123123", "name": "Test"}
+        self.parser = RitzauFeedParser()
+
+        with open(fixture, "rb") as f:
+            self.xml = f.read()
+            self.xml_root = etree.fromstring(self.xml)
+            self.item = self.parser.parse(self.xml_root, provider)
+
+
+class RitzauTestCase(BaseRitzauTestCase):
+
+    # def test_can_parse(self):
+    #     self.assertTrue(RitzauFeedParser().can_parse(self.xml_root))
+
+    def test_content(self):
+        item = self.item
+        print(item)
+        self.assertEqual(item["anpa_category"], [{"name": "Udland", "qcode": "udland"}])
```

Comment:
I think the idea was to populate it based on `categories` CV and not to introduce new values.
so we need to add to categories CV a field like `ritzau_section_id` and if we find a match we should populate it, eg.:
```
{"_id": "categories", "items": [
    {"name": "Foo", "qcode": "foo", "ritzau_section_id": "2"}
]}
```
and in that file it would populate `anpa_category` to `{"name": "Foo", ...}`

Reply:
and we have to check `ritzau_section_id` with this xml `section_value` eg: `Udland`